### ISSUE TITLE
Where?

### ISSUE THREAD

**Issue by fmarzocca** [salviador/SCS-Raspberry-pi#2]
I didn't understand how/where can I get the SCS-shield...

**Comment by salviador** [comment]
yes, it is in the testing process, I expect between 1 - 2 months to complete it

**Comment by fmarzocca** [comment]
Great. I will keep subscribed to this thread, waiting for your tests and
announcements.

**Comment by fmarzocca** [comment]
Will the shield be able to act as a sort of gateway to the SCS? I mean, could it be used to send commands and monitor the bus?

**Comment by salviador** [comment]
sure you can sniff the bus just by doing a

mosquitto_sub -h localhost -t "/scsshield/debug/bus"

**Comment by efa** [comment]
I,
I found the sites:
http://scsshields.altervista.org/index.html
https://github.com/salviador/SCS-Raspberry-pi

It's been a long time since the last updates.
Don't let the project die, publish schematics and pcb masters on github, so that others can participate, help, test, contribute

Thank you,
--
Valerio

**Comment by Saruspete** [comment]
Hello
I see the project looks dead, but I would too love to have the schematics to use a raspberry to interact with the SCS bus too !
There was other projects to debug/decode the bus ( https://michael.stapelberg.ch/posts/2020-09-28-nuki-scs-bticino-decoding/#capturing-the-scs-bus-with-sigrok ) but having a rpi expansion module would be easier :)

Note: After searching more, the hardware is [sold on ebay](https://www.ebay.it/itm/126471866142?customid=&toolid=10050)

Thanks a lot !
